Tools/autotest/param_metadata/param_parse.py

```diff
@@ -111,9 +114,6 @@ def error(str):
             path = path.strip()
             debug("\n Processing file '%s'" % path)
             if path.endswith('.pde') or path.find('/') == -1:
-                if len(vehicles) != 1:
-                    print("Unable to handle multiple vehicles with .pde library")
-                    continue
```

Comment:
Removing this without any other change doesn't seem right. The _libraryfname_ will always use the first vehicle but that might be the wrong vehicle.
And actually there's another problem when finding libraries: we are checking for equal names instead of path. That means that, for example, when checking all vehicles, we get RALLY from Plane but not from Copter (and they have different Path values).
I think that we either properly support checking for all vehicles at once or we only support checking for one vehicle at a time.